### DISCUSSION TITLE
[Question][Hw5] About exampleX.expected file

### DISCUSSION THREAD

**Discussion by Quasar0311** [prosyslab-classroom/cs348-information-security#603] [Q&A]
I checkout the skeleton code before fixing test/dune file, and I saw the announcement that there is a change in test/dune file. However, there is such line (diff example4.expected example4.output), which needs example4.expected file. However, my repo doesn't have example 4 & 5's expected files. Should I have to re-checkout the skeleton code? Or where can I get those files?

**Comment by RiceBiscuits** [comment]
You can find content of the files below.

https://github.com/prosyslab-classroom/cs448-2021-spring/issues/130#issuecomment-846989010

**Comment by Quasar0311** [comment]
Thanks!
